rackcity/views/asset_views.py

```diff
@@ -137,8 +143,24 @@ def asset_detail(request, id):
     return JsonResponse(serializer.data, status=HTTPStatus.OK)
 
 
+def get_change_plan(change_plan_id):
+    change_plan = ChangePlan.objects.get(
+        id=change_plan_id
+        )
```

Comment:
this needs to be wrapped in a try/except with ObjectDoesNotExist - it will throw an uncaught error if there is no change plan with the matching id